components/auth/form/signup-form.tsx

```diff
@@ -24,16 +24,45 @@ const SignupForm = () => {
 
   const handleInputChange = (e: ChangeEvent<HTMLInputElement>) => {
     const { name, value } = e.target;
-
-    console.log(value);
-
     setFormData((prev) => {
       return { ...prev, [name]: value };
     });
   };
 
-  const handleSubmit = () => {
-    console.log('Form submitted:', formData);
+  const handleSubmit = async () => {
+    try {
+      const payload: TSignupPayload = {
+        company_name: formData.OrganizationName,
+        user_info: {
+          name: formData.FullName,
+          email: formData.Email,
+          password: formData.Password,
+        },
+      };
+
+      console.log(payload);
+
+      const apiBaseUrl = process.env.API_BASE_URL;
```

Comment:
Not this way Brother. Checkout the `auth-services.ts` file into `services` folder. We have to create an instance for `AuthService` at the top level of this component and use the `signup()` method here.

The instance creation here will looks like:
```javascript
const AuthServices = new AuthService();
```

Later we have to use it here at the following way:
```javascript
const response = await AuthServices.signup(payload);
```